integration/js/script/run-test

```diff
@@ -62,6 +62,12 @@ case $1 in
       r app_quit_audio_test.config.json
       r meeting_leave_audio_test.config.json
       ;;
+   video_simulcast)
+      r video_test_simulcast.config.json
+      r app_quit_video_test.config.json
```

Comment:
The following 3 tests are already run in the video group right? The new simulcast should go to the other group too.